src/Components/Components/src/Auth/AuthorizeViewCore.razor

```diff
@@ -1,4 +1,8 @@
-@namespace Microsoft.AspNetCore.Components
+@*
+    Ideally, this would be an internal (and probably abstract) class. So temporarily
+    this is in an "internal" namespace and should become actually internal in the future.
+*@
+@namespace Microsoft.AspNetCore.Components.Internal
```

Comment:
Is this is causing us so much "grief". Why don't we author this as a plain c# class here? The markup is simple and avoids all the problems you mention.

Reply:
You're right - this is better as a plain C# class. Done, and removed the pubternalness since it's now `abstract`.